app/src/main/java/com/github/fi3te/notificationcron/data/local/NotificationCronInput.kt

```diff
@@ -0,0 +1,8 @@
+package com.github.fi3te.notificationcron.data.local
+
+data class NotificationCronInput (
```

Comment:
Reformat file (indentation)

Reply:
The double indent is actually what I get on reformatting. 8 spaces for continuation indent seem to be the default settings in Android studio.

Reply:
Fixed.